app/serializers/book_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class BookSerializer < ActiveModel::Serializer
+  attributes :title, :author, :genre, :publisher, :year
```

Comment:
And the image? Also the doc ask for created_at and updated_at.

Reply:
I keep forgetting the image exists and I didn't realize those were necessary too. I'll add them now.